playbook/setupNode.yml

```
@@ -0,0 +1,9 @@
+---
+- hosts: all
+  tasks:
+    - name: get avalanche go script
+      shell: wget -nd -m https://raw.githubusercontent.com/ava-labs/avalanche-docs/master/scripts/avalanchego-installer.sh
+    - name: modify permissions
+      shell: chmod 755 avalanchego-installer.sh
+    - name: call avalanche go install script
+      shell: ./avalanchego-installer.sh --ip static --rpc private --state-sync on --fuji
```

```suggestion
      command: ./avalanchego-installer.sh --ip static --rpc private --state-sync on --fuji
```
Same reason as above